sdk/communication/Azure.Communication.Chat/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## 1.0.0-beta.6 (Unreleased)
 
+### Added
+- Updated client constructors URL variable to `endpoint`.
+- Renamed `ChatThread` model to `ChatThreadProperties`.
+- Renamed `GetChatThread` operation to `GetPropertie`s and moved it to `ChatThreadClient`.
+- Made list of participants optional for `CreateChatThread`.
+- Renamed `ChatThreadInfo` model to `ChatThreadItem`.
```

Comment:
Things that are renamed or moved should be in the a "Breaking Change" subsection instead of "Added"

Reply:
moved :)